boards/arm/mimxrt1064_evk/mimxrt1064_evk.dts

```diff
@@ -54,7 +54,7 @@
 		compatible = "pwm-leds";
 
 		green_pwm_led: green_pwm_led {
-			pwms = <&flexpwm2_pwm3 0 PWM_MSEC(20)>;
+			pwms = <&flexpwm2_pwm3 0 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
```

Comment:
You need to update the other mimxrt10xx_evk boards as well.

Reply:
Fixed